scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableSyntaxConfig.scala

```diff
@@ -13,6 +13,7 @@ case class DisableSyntaxConfig(
     noSemicolons: Boolean = false,
     noTabs: Boolean = false,
     noXml: Boolean = false,
+    noVariantTypes: Boolean = false,
```

Comment:
Can we split this into `noContravariantTypes` and `noCovariantTypes`? I can imagine situation where one would like to disable one but not the other.